src/inputs/checkbox.scss

```diff
@@ -58,5 +58,11 @@
       opacity: 1;
       transition: opacity $iui-speed-fast ease;
     }
+
+    input:enabled ~ .iui-checkbox-checkmark > svg > .iui-uncheck {
```

Comment:
You don't need `:enabled` here, as disabled checkboxes already have a high specificity which should override this if you remove `> svg`.

![image](https://user-images.githubusercontent.com/9084735/143048144-b418a5e7-bf9b-4b45-a1cb-e9c223c84ba4.png)

But I did have to reduce specificity in one other place in `checkbox-radio`.
![image](https://user-images.githubusercontent.com/9084735/143048361-18c8f08c-5a4c-440e-ad5d-12102ecf11be.png)

Reply:
Great suggestion, change is made!